syntax = "proto3";
package app.protobuf;
option go_package=".;proto";

message ChatRoomData {
  message ChatRoomMember {
      string wxID = 1;
      string displayName = 2;
      int32 state = 3;
  }
  repeated ChatRoomMember members = 1;
  int32 field_2 = 2;
  int32 field_3 = 3;
  int32 field_4 = 4;
  int32 room_capacity = 5;
  int32 field_6 = 6;
  int64 field_7 = 7;
  int64 field_8 = 8;
}